jupyterhub/templates/pod-culler/rbac.yaml

```diff
@@ -1,42 +0,0 @@
-{{- if and .Values.cull.enabled .Values.cull.maxAge }}
-{{- if .Values.rbac.enabled -}}
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: pod-culler
-  labels:
-    {{- include "jupyterhub.labels" . | nindent 4 }}
----
```

Comment:
This serviceAccount, and associated role bound to it, was only used by the pod-culler deployment.